Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.ts

```diff
@@ -124,40 +126,17 @@ export class ProfilePictureComponent implements OnInit, OnDestroy {
     if (this.canEnlargeOnClick === false || this.dialogRef != null || this.imageLoaded === false) {
       return;
     }
-    this.openEnlarge(true);
-  }
-
-  public onMouseleave() {
```

Comment:
removing this will cause the dialog to not close automatically in the table view